README.md

```diff
@@ -27,14 +27,10 @@ which has been tested to use from this exporter:
  - `0.10.0.1`
  - `0.9.0.1`
 
-Build
-=====
+Build & install
```

Comment:
I meant to have two sections `Install` and `Build` separately.
When I started touching go, I often struggle to getting start developing an existing project because of variety of build processes.
I thought it'd be better to leave the original instructions for people who wants to patch this project. WDYT?